addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.ActivityMarkDonePopover" owl="1">
         <div class="o_ActivityMarkDonePopover" t-on-keydown="_onKeydown">
             <t t-if="activity">
-                <textarea class="form-control o_ActivityMarkDonePopover_feedback" rows="3" placeholder="Write Feedback" t-ref="feedbackTextarea"/>
+                <textarea class="form-control o_ActivityMarkDonePopover_feedback" rows="3" placeholder="Write Feedback" t-on-blur="_onBlur()" t-ref="feedbackTextarea"/>
```

Comment:
usually we tell the method here so just `_onBlur`, not actually calling it (though OWL supports both syntax)